app/authentication/auth.py

```diff
@@ -50,14 +62,17 @@ def get_auth_token(req):
         raise AuthError("Unauthorized, authentication token must be provided", 401)
 
     for el in AUTH_TYPES:
-        scheme, auth_type = el
+        scheme, auth_type, _ = el
         if auth_header.lower().startswith(scheme.lower()):
             token = auth_header[len(scheme) + 1 :]
             return auth_type, token
 
-    # TODO: decide if error message stays the same.
-    # In fact we support another auth method than Bearer
-    raise AuthError("Unauthorized, authentication bearer scheme must be used", 401)
+    raise AuthError(
+        "Unauthorized, Authorization header is invalid. "
```

Comment:
Is `Unauthorized` necessary here? Consider `Authorization header is invalid.`
Would this documentation be helpful to point people to in this case? https://documentation.notification.canada.ca/en/start.html#headers